### ISSUE TITLE
Error in colMeans(x, na.rm = TRUE) : 'x' must be numeric

### ISSUE THREAD

**Issue by Sibusiso1995** [facebookexperimental/Robyn#431]
Hi there, I've been experiencing this issue even though I change my data types to numeric when I view the dataset again it still shows <dbl> and doesn't change to numeric.
<img width="773" alt="Screenshot 2022-07-18 at 12 58 50" src="https://user-images.githubusercontent.com/54172627/179497577-25224228-623a-4121-8286-1dbc3d8e4bd8.png">

**Comment by kyletgoldberg** [comment]
i don't think so, but could you try saving as CSV and uploading that way?

**Comment by Sibusiso1995** [comment]
I've tried that and encountered the same problem

**Comment by Sibusiso1995** [comment]
I do for one column that contains 0 only

**Comment by kyletgoldberg** [comment]
can you remove that column if it's only 0s?

**Comment by Sibusiso1995** [comment]
Still nothing

**Comment by kyletgoldberg** [comment]
okay would you mind sending me the data file and your code so i can debug on my end? kylegoldberg@fb.com

**Comment by Sibusiso1995** [comment]
Let me try again from the top till the point of building the initial model
